From d139474f2a6d87c1cd20dfbb916fe650bfcce968 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Tue, 4 Jun 2019 11:39:39 +0800
Subject: [PATCH] multipath-tools: modify Makefile.inc for cross-compilation

Do not look for systemd info on the host, and allow us to pass in CFLAGS
using the OPTFLAGS variable.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Joe Slater <joe.slater@windriver.com>

Update for version 0.5.0-144-g770e6d0

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Update for version 0.7.1

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

update for version 0.7.7

remove change about CFLAGS part, since patch 0024 already have similar function.

Signed-off-by: Changqing Li <changqing.li@windriver.com>

update for version 0.8.1
Signed-off-by: Changqing Li <changqing.li@windriver.com>
[OP: Rebase to 0.9.3]
Signed-off-by: Ovidiu Panait <ovidiu.panait@windriver.com>
---
 Makefile.inc | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/Makefile.inc b/Makefile.inc
index 34f2cc6d..ee6a4c9f 100644
--- a/Makefile.inc
+++ b/Makefile.inc
@@ -42,17 +42,6 @@ ifndef RUN
 	endif
 endif
 
-ifndef SYSTEMD
-	ifeq ($(shell $(PKGCONFIG) --modversion libsystemd >/dev/null 2>&1 && echo 1), 1)
-		SYSTEMD = $(shell $(PKGCONFIG) --modversion libsystemd | awk '{print $$1}')
-	else
-		ifeq ($(shell systemctl --version >/dev/null 2>&1 && echo 1), 1)
-			SYSTEMD = $(shell systemctl --version 2> /dev/null | \
-				sed -n 's/systemd \([0-9]*\).*/\1/p')
-		endif
-	endif
-endif
-
 ifndef SYSTEMDPATH
 	SYSTEMDPATH=usr/lib
 endif
-- 
2.38.1

